### ISSUE TITLE
Error when using this with NUnit

### ISSUE THREAD

**Issue by jesperlandmer** [lucaslorentz/minicover#3]
Getting this error when using minicover with nunit

```
`Exception System.InvalidOperationException, Exception thrown executing tests in /Users/Jesper/2dv610/Snake-Game/bin/Debug/netcoreapp2.0/Snake-Game.dll
Operation is not valid due to the current state of the object.
   at Mono.Cecil.ModuleDefinition.ReadSymbols(ISymbolReader reader)
   at Mono.Cecil.ModuleReader.ReadSymbols(ModuleDefinition module, ReaderParameters parameters)
   at Mono.Cecil.ModuleReader.CreateModule(Image image, ReaderParameters parameters)
   at Mono.Cecil.ModuleDefinition.ReadModule(String fileName, ReaderParameters parameters)
   at NUnit.VisualStudio.TestAdapter.NavigationDataProvider.CacheNewTypes(String assemblyPath, IDictionary`2 types) in C:\Users\Terje\source\repos\nunit\nunit3-vs-adapter\src\NUnitTestAdapter\NavigationDataProvider.cs:line 85
   at NUnit.VisualStudio.TestAdapter.NavigationDataProvider.GetNavigationData(String className, String methodName) in C:\Users\Terje\source\repos\nunit\nunit3-vs-adapter\src\NUnitTestAdapter\NavigationDataProvider.cs:line 48
   at NUnit.VisualStudio.TestAdapter.TestConverter.MakeTestCaseFromXmlNode(XmlNode testNode) in C:\Users\Terje\source\repos\nunit\nunit3-vs-adapter\src\NUnitTestAdapter\TestConverter.cs:line 144
   at NUnit.VisualStudio.TestAdapter.TestConverter.ConvertTestCase(XmlNode testNode) in C:\Users\Terje\source\repos\nunit\nunit3-vs-adapter\src\NUnitTestAdapter\TestConverter.cs:line 79
   at NUnit.VisualStudio.TestAdapter.NUnit3TestExecutor.RunAssembly(String assemblyPath, TestFilter filter) in C:\Users\Terje\source\repos\nunit\nunit3-vs-adapter\src\NUnitTestAdapter\NUnit3TestExecutor.cs:line 249
NUnit Adapter 3.9.0.0: Test execution complete
No test is available in /Users/Jesper/2dv610/Snake-Game/bin/Debug/netcoreapp2.0/Snake-Game.dll. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.`
```

Any idea what the issue might be?

**Comment by lucaslorentz** [comment]
Hey @jesperlandmer, thanks for trying it.

I will check it with Nunit when I get some time, if you get some time to setup a very small repository reproducing the issue it would be very helpful.

**Comment by lucaslorentz** [comment]
@jesperlandmer 

It should be fixed on latest version: 2.0.0-ci-20180101105210

Problem was that I was not saving the updated pdb file. Then it was impossible for NUnit to read assembly + symbols again using Cecil.

After latest changes I was able to run it with Nunit inside Visual Studio and also debug with breakpoints.

**Comment by jesperlandmer** [comment]
Okay, great, thanks! Keep up the great work!
